examples/blackscholes/blackscholes_pa.py

```diff
@@ -0,0 +1,55 @@
+import numba
+import numpy as np
+# from scipy.special import erf
```

Comment:
Done. For future reference, this blackscholes code would be simpler if `scipy.special.erf` or `scipy.stats.norm.cdf` was supported by Numba.